app/src/main/java/org/mozilla/focus/session/SessionManager.java

```diff
@@ -71,6 +71,11 @@ private void createSessionFromIntent(Context context, SafeIntent intent) {
         final String action = intent.getAction();
 
         if (Intent.ACTION_VIEW.equals(action)) {
+            final String dataString = intent.getDataString();
+            if (TextUtils.isEmpty(dataString)) {
```

Comment:
From the code, it looks like the behavior is: if we receive a null URI, we open the app and do nothing. Should we should a Toast for the users to let them know the Intent failed?